### DISCUSSION TITLE
How can I convert a GAUSSIAN file

### DISCUSSION THREAD

**Discussion by ShaochengShen** [deepmodeling/dpdata#243] [Q&A]
Dear developers，
    In the process of using dpdata, I wanted to convert GAUSSIAN files (i.e., .out files) to .RAW files and .npy files, but I could not convert them according to the manual format. I was very confused, so I wanted to ask how to convert them, or whether they can be converted?
The .out file and related commands and error reports are attached below.

[Dopamin2.txt](https://github.com/deepmodeling/dpdata/files/8037818/Dopamin2.txt
![Screenshot from 2022-02-10 11-32-10](https://user-images.githubusercontent.com/94623045/153332377-2addf703-8119-4d42-9f78-939b2cdbb24d.png)
![Screenshot from 2022-02-10 11-32-52](https://user-images.githubusercontent.com/94623045/153332352-132a643c-f90e-486b-b220-d95d95d8b8dd.png)

**Comment by wanghan-iapcm** [comment]
Could you please provide a minimal example `.out` file and the steps you used for conversion? @njzjz should be able to help.

**Reply by ShaochengShen** [reply]
Thanks for your suggestion!

**Comment by njzjz** [comment]
1. forces are not printed;
2. input orientation is not printed.

**Reply by ShaochengShen** [reply]
In the output process, I found that the. out file could not read out the corresponding coordinates, forces and so on. I think this may be the reason for the incompatibility of the file format.Hope you can be more detailed, thanks a lot.

**Reply by njzjz** [reply]
I don't find forces in the file you provided.

**Comment by ShaochengShen** [comment]
Maybe there is something wrong with my idea,which needs time to think.Thanks a lot!



---Original---
From: "Jinzhe ***@***.***&gt;
Date: Fri, Feb 11, 2022 02:15 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [deepmodeling/dpdata] How can I convert a GAUSSIAN file(Discussion #243)




 
I don't find forces in the file you provide.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
